cmd/quickPrimateRecon/quickPrimateRecon.go

```diff
@@ -130,5 +41,5 @@ func main() {
 	inFile := flag.Arg(0)
 	outFile := flag.Arg(1)
 
-	quickPrimateRecon(inFile, outFile)
+	quickPrimateRecon(inFile, outFile, *messyToN)
```

Comment:
It seems like this function is expecting the infile to have primate sequences in a specific order based on how they are being assigned to variables (looks like human -> bonobo -> chimp -> orangutan -> gorilla). If this is true, it might be worth putting this in the usage message.

Reply:
This is a good point.  Not needed for this checkin, but maybe a more robust way would be to pass in a map of fasta sequences keyed on the name of the fasta record that is expected to be certain things "human", "chimp", etc?